pkg/analysis_server/lib/src/services/correction/fix.dart

```diff
@@ -189,6 +190,8 @@ class DartFixKind {
       'LINT_REMOVE_INTERPOLATION_BRACES',
       50,
       'Remove unnecessary interpolation braces');
+  static const LINT_REPLACE_WITH_COLLECTION_LITERAL = const FixKind(
+      'LINT_REPLACE_WITH_COLLECTION_LITERAL', 50, 'Replace with literals');
```

Comment:
Please remove the `LINT_` prefix. (I know the two existing fixes were named that way, but they shouldn't have been, and I'll fix them eventually.) Also, replace "literals" with "literal".